jitbuilder/lljb/include/lljb/Compiler.hpp

```diff
@@ -0,0 +1,120 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
```

Comment:
Why does the include directory have `lljb` as a subdirectory (since it already lives under a `lljb` directory)?  Sorry, I think I asked you this before a few months back but I forgot...

Reply:
It's just one way that many C++ projects (including llvm and clang) structure header files. This way, when specifying header directory paths for a project, you need to just add `project/include` folder without worrying about the order you are adding the paths. When including header files, you can be specific about exactly which header file you are using by specifying `include "lljb/MethodBuilder"` in your code file without having to worry about how the header search paths are ordered by the build system, removing the potential for including some other project's `MethodBuilder` file. Since some lljb's files have shared names with what exists in OMR already, it was simpler to structure the include directory this way.